lib/request_code.dart

```diff
@@ -34,6 +34,9 @@ class RequestCode {
     await controller.setBackgroundColor(Colors.transparent);
     await controller.setUserAgent(_config.userAgent);
     await controller.loadRequest(launchUri);
+    await controller.setNavigationDelegate(NavigationDelegate(
```

Comment:
Hi  @tajjacob,

This is not working when onPageFinished function is null. Can you take a look at it please?

Reply:
sure. I'll let you know once I fix it.

Reply:
@josemiguelvarela I added this lines to solve the issue. Please check if its working or not

```
    if (_config.onPageFinished != null) {
      await controller.setNavigationDelegate(
        NavigationDelegate(
          onPageFinished: _config.onPageFinished,
        ),
      );
    }
```

Reply:
Sorry for the late reply @tajjacob . I'll review it today, thanks!